sioux/core/text_annotation.py

```diff
@@ -21,6 +21,11 @@ def __init__(self, json_str=None, pipeline_instance=None):
         self.pipeline = pipeline_instance
 
         self.text = result_json["text"]
+        if len(self.text) <= 0:
```

Comment:
would it be better to change it on the line above? `self.text = result_json["text"].strip()`